apps/web/lib/app-providers.tsx

```diff
@@ -302,7 +305,16 @@ const AppProviders = (props: AppPropsWithChildren) => {
                 <OrgBrandProvider>
                   {/* @ts-expect-error FIXME remove this comment when upgrading typescript to v5 */}
                   <CacheProvider>
-                    <MetaProvider>{props.children}</MetaProvider>
+                    <MetaProvider>
```

Comment:
Can you make the same change to `app-providers-app-dir.tsx`?

Reply:
Update: I'll limit `DubAnlaytics` to just `/app/web/pages/signup.tsx` since that's the only page we need this script on – this should simplify things quite a bit!